### ISSUE TITLE
Is this usable from macOS?

### ISSUE THREAD

**Issue by NightMachinary** [marin-m/SongRec#96]
Is this usable from macOS? 

The CLI at least should work? Though I can't find any docs on the CLI.

**Comment by msdos** [comment]
Yes. Check https://github.com/marin-m/SongRec/issues/91 for some information about needed libs for compiling using cargo.

**Comment by martin-braun** [comment]
@msdos Would you feel confident to publish the build on brew.sh?
